### DISCUSSION TITLE
20240725 Weekly Development Meeting

### DISCUSSION THREAD

**Discussion by cgbosse** [nautilus-cyberneering/.github#103] [Meetings]
- Time:  9:30 in Canary Islands (WET (UTC+0)).
- Time: 10:30 in Vaduz (CET (UTC+1)).

Location:
<https://meeting.blockfinance-eco.li/NautilusCyberneeringWeeklyDevMeeting>

Expected duration: 2 to 4 hours with breaks

Communicated attendants:

- Cameron Garnham(@da2ce7)  - 
- Jose Celano(@josecelano) - 
- Constantin Bosse(@cgbosse) - x
- Mario Melchor(@mario-nt) - 
- Graeme Byrne(@grmbyrn) - x

-------------------------------------------------------------

## Agenda:

1. Everyone will take their turn and:
- Go over his notes added as comments for the meeting which will have been added in advance to give an update of all the work done during the previous week indicating.
	- The issues worked on
	- The progress made on them
	- Other things done
- Inform of any faced challenges and request help if necessary to work on specific issues.
- Highlight the encountered difficulties, especially if there is a risk of them causing delays or affecting end product's quality.
- Communicate what will be the next week's tasks.
- List any new topics for further discussion in an additional Technical meeting that he noted for future discussion in comments to this discussion either before the meeting or after his turn so that they get considered.

During the meeting every team member will take their turns and follow this sequence.
The topics for further discussion will be noted in advance by each the team member and to the meetings agenda.

2. At the end of the meeting if there any new technical topics to be discussed further it will be decided to
set another meeting date for the discussion of these or to hold the meeting hereafter.

Also the persons to be part of the meeting will be decided on so that the others will be free to continue their own work.

**Comment by grmbyrn** [comment]
## Torrust Website
* Add `(blog-article)` route for individual blog posts
* Create `(pages)` route to group together pages such as `tracker`, `index`, `about`, etc.
* Accordion component which is currently being used to display or hide sublists in Table of Contents in each route in `(pages)`
* Fix missing date error in `BlogPosts.svelte`
* Add content for pages from Miro board
* Fix `Accordion` component as all sublists were being displayed when an `Accordion` was clicked. Now only the sublist targeted by the clicked `Accordion` appears or disappears.
* Add `constants` folder to contain info which can be imported into the component/page where it is required.
* Fix incorrect id for `whyTorrust` in `About`
* Fix linting errors
* Rebase commits into single commit
